### PR TITLE
Wellfound Rebrand, Tag Fix

### LINKED ISSUE #365: Tag with icon vertical alignment
**Unrelated:** something odd is going on with vertical alignment. I wonder what's causing consecutive Tags to bump up/down.

valignment

_Originally posted by @drewlustro in https://github.com/venturehacks/halo/pull/364#pullrequestreview-1013956107_

### PR DESCRIPTION
**Pull request by doremidom**
Fixes #365 

Adds new brand fonts and removes old ones

Updates the logo and copy to Wellfound
Screen Shot 2023-01-18 at 10 29 23 AM
Screen Shot 2023-01-18 at 10 29 28 AM
Fixed tag alignment:
Screen Shot 2023-01-18 at 10 55 19 AM

### CODE REVIEW

File: packages/halo/src/components/core/Tag/index.tsx [hunk 1]
```diff
@@ -98,8 +98,9 @@ function TagRaw({
       {...onClickProps}
       {...rest}
     >
-      {icon && <span className={iconClassnames}>{icon}</span>}
```
Comment (vishaljeet) [inline packages/halo/src/components/core/Tag/index.tsx:101]: QQ: how come we don't need to adjust the size here anymore?

File: packages/halo/src/pages/welcome.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { Button } from '../components/core/Button';
     components, enabling product teams to ship rapidly.
   </div>
   <div className="flex-none">
-    <a target="_blank" href="https://buildkite.com/angellist/halo">
```
Comment (vishaljeet) [inline packages/halo/src/pages/welcome.mdx:20]: I think this should remain as-is?

**Review by vishaljeet — APPROVED** [review]
LG!

File: packages/halo/src/components/core/Tag/index.tsx (hunk 1 above)
Comment (doremidom) [reply packages/halo/src/components/core/Tag/index.tsx:101]: was an oversight when trying to fix alignment, will add back

**Pull request merged**